_posts/events/2019-09-27-mumttt.md

```diff
@@ -0,0 +1,77 @@
+
+TOPICS OF INTEREST
+
+The MUMTTT 2019 workshop invites the submission of papers reporting on original and unpublished research on topics related to MWU processing in machine translation and translation technology, including:
+ * Lexical, syntactic, semantic and translational aspects in MWU representation
+ * Theoretical approaches to MWUs (e.g., collostructional analysis of MWU, cognitive approaches to processing MWUs, etc.)
+ * Development of multilingual MWU resources
+ * Identification and acquisition of MWUs and variants
+ * Learning semantic information about MWUs from monolingual, parallel or comparable corpora
+ * Development and use of MWU resources in machine translation and translation technology
+ * Development of corpora for the extraction and translation of MWUs
+ * Compilation of resources for the extraction and translation of multiword units
+ * Creation of MWU-annotated corpora with a focus on translation aspects
+ * Paraphrasing of MWUs applied to the improvement of machine translation
+ * MWUs and word alignment techniques
+ * MWUs in machine translation
+ * MWUs in translation memory systems
+ * MWUs in term extraction
+ * MWU-centred machine translation evaluation
+ * Evaluation of MWU translation
+ * MWUs in CAT tools
+ * Multilingualism and MWU processing
+ * Psycholinguistic studies of MWU processing in a bilingual setting.
+
+SUBMISSION AND PUBLICATION
+
+Submissions must consist of full-text papers and should not exceed 7 pages excluding references.
+
+A selection of papers will be invited to submit an extended version for the second volume of Multiword Units in Machine Translation and Translation Technology to be published by John Benjamins by the end of 2019.
+
+They must be formatted according to the ACL 2017 style guidelines available both for Word and LaTeX text processor. Each submission will be reviewed by at least three programme committee members. Accepted papers will be presented orally or as posters, as determined by the programme committee. There will be no distinction in the workshop proceedings between papers presented orally or as posters. The proceedings will be published both as e-proceedings with ISSN and as an electronic volume with ISBN and will be made available at the time of the conference. 
+
+All submissions must be made online using the Softconf START conference management system:   https://www.softconf.com/j/mumttt2019.
```

Comment:
Put `<` and `>` around the URL so that it is rendered as a link.

Reply:
Done